examples/Applepay/index.php

```diff
@@ -0,0 +1,240 @@
+
+            onMerchantValidationCallback: (session, event) => {
+                $.post('./merchantvalidation.php', JSON.stringify({"merchantValidationUrl": event.validationURL}), null, 'json')
+                    .done(function (validationResponse) {
+                        // Contains Applepay session on succsess.
+                        session.completeMerchantValidation(validationResponse);
+                    })
+                .fail(function (error) {
+                        handleError(JSON.stringify(error.statusText));
+                        session.abort();
+                    }
+                )
+            },
+
+            onPaymentAuthorizedCallBack: (session, event) => {
+                var paymentData = event.payment.token.paymentData;
+                const $form = $('form[id="payment-form"]');
+                let formObject = QueryStringToObject($form.serialize());
+
+                unzerApplePayInstance.createResource(paymentData)
+                    .then(function (createdResource) {
+                        formObject.typeId = createdResource.id;
+                        // Hand over the type ID to your backend.
+                        $.post('./Controller.php', JSON.stringify(formObject), null, 'json')
+                            .done(function (result) {
+                                // Handle the transaction respone from backend.
+                                let status = result.transactionStatus;
+                                if (status === 'success' || status === 'pending') {
+                                    session.completePayment({status: window.ApplePaySession.STATUS_SUCCESS});
+                                    window.location.href = '<?php echo RETURN_CONTROLLER_URL; ?>';
+                                } else {
+                                    window.location.href = '<?php echo FAILURE_URL; ?>';
+                                    abortPaymentSession(session);
+                                    session.abort();
+                                }
+                            })
+                            .fail(function (error) {
+                                handleError(error.statusText);
+                                abortPaymentSession(session);
+                            });
+                    })
+                    .catch(function (error) {
+                        handleError(error.message);
+                        abortPaymentSession(session);
+                    })
+            },
+
+            onShippingMethodSelectedCallback: (session, event) => {
+                var status = ApplePaySession.STATUS_SUCCESS;
+                var newTotal = {
+                    'label': 'Total amount',
+                    'amount': '12.99',
+                    'type': 'final'
+                }
+                var newLineItems =[]; // What does this mean?
+                session.completeShippingMethodSelection(status, newTotal, newLineItems);
```

Comment:
Maybe remove the comment, or just do the call without the newLineItems parameter
Here you can find, what it is for: https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession/1778024-completeshippingmethodselection

Reply:
I don't think we need the `newLineItems` in the examples at all

Reply:
removed it for now.